app/elements/coder/code-editor/file-explorer/file-explorer.html

```diff
@@ -44,15 +48,16 @@
 				'file-selected': '_fileSelected'
 			},
 			ready: function(){
+				this.rootItem = {name: '/', value: {}}
```

Comment:
`this.rootItem` violates constraints on a file name. A file name must not contain `/`.